language-support/ts/codegen/tests/src/DA/Test/Daml2Ts.hs

```diff
@@ -333,3 +292,7 @@ tests damlTypes yarn damlc daml2ts davl = testGroup "daml2ts tests"
     assertFileLines file expectedContent = do
       actualContent <- T.lines <$> T.readFileUtf8 file
       assertEqual ("The content of file '" ++ file ++ "' does not match") expectedContent actualContent
+
+    assertIsInfixOf :: String -> String -> String -> IO ()
```

Comment:
We already have `assertInfixOf` in `DA.Test.Util`. Maybe just add a `msg argument to that instead of duplicating it here?

Reply:
Good catch. Fixed in https://github.com/digital-asset/daml/pull/5245.